src/pages/EditPage.js

```diff
@@ -17,42 +18,62 @@ import LooMap from '../components/LooMap';
 import Box from '../components/Box';
 
 import useNearbyLoos from '../components/useNearbyLoos';
-import useMapPosition from '../components/useMapPosition';
+import { useMapState } from '../components/MapState';
 
 import config from '../config';
 import history from '../history';
 
-const FIND_BY_ID = loader('./findLooById.graphql');
 const UPDATE_LOO = loader('./updateLoo.graphql');
 
 const EditPage = (props) => {
-  // find the raw loo data for the given loo
-  const { loading: loadingLooData, data: looData, error: looError } = useQuery(
-    FIND_BY_ID,
-    {
-      variables: {
-        id: props.match.params.id,
-      },
-    }
+  const { loading: loadingLooData, data: looData, error: looError } = useSWR(
+    `{
+      loo(id: "${props.match.params.id}") {
```

Comment:
This makes loads of alarms ring in my head.

Can we pass variables? https://github.com/prisma-labs/graphql-request#using-variables

Reply:
yup i think that looks nicer, although the results are the same; a unique cache key for each find loo by id query

Reply:
For me it triggered old "passing user input from the URL into a database query by concatenation" reflexes

Reply:
aha makes sense